src/components/AnonymousChat/child/ChatItemTargetText.tsx

```diff
@@ -14,6 +14,7 @@ import {
 } from './ChatItemAttachmentStyles';
 import ChatItemAttachment from './ChatItemAttachment';
 import dimen from '../../../utils/dimen';
+import {LinkableText} from '../../LinkableText';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `LinkableText` is correctly added. However, it's a good practice to keep imports from the same directory or similar functionality together. This helps in maintaining readability and understanding the dependencies of the module.
```diff
  import ChatItemAttachment from './ChatItemAttachment';
  import dimen from '../../../utils/dimen';
+ import {LinkableText} from '../../LinkableText';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->